### PR TITLE
Refactored printing/pretty/pretty.py to use pretty_symbology.py

### PR DESCRIPTION
**Pull request by ShikharJ**
Fixes: #10983 
Please review and suggest improvements.

### CODE REVIEW

**Comment by ShikharJ** [comment]
Ping @asmeurer.

File: sympy/printing/pretty/pretty_symbology.py [hunk 1]
```diff
@@ -580,3 +580,36 @@ def annotated(letter):
...
+     'ElementOf' :                             u" \N{ELEMENT OF} ",
+     'DoubleStruckItalicSmallD' :              u"\N{DOUBLE-STRUCK ITALIC SMALL D}",
+     'RightArrowFromBar' :                     u" \N{RIGHTWARDS ARROW FROM BAR} ",
+     'BoxDrawingsLightDownAndHorizontal' :     u'\N{BOX DRAWINGS LIGHT DOWN AND HORIZONTAL}',
+     'SuperscriptPlusSign' :                   u'\N{SUPERSCRIPT PLUS SIGN}',
+     'SuperscriptMinus' :                      u'\N{SUPERSCRIPT MINUS}',
+     'Dagger' :                                u'\N{DAGGER}',
+     'RightParenthesisUpperHook':              u'\N{RIGHT PARENTHESIS UPPER HOOK}',
+     'SquareRoot' :                            u'\N{SQUARE ROOT}',
+     'DoubleStruckCapitalZ%d' :                u'\N{DOUBLE-STRUCK CAPITAL Z}_%d',
```
Comment (asmeurer) [inline sympy/printing/pretty/pretty_symbology.py]: The %d part should be done separately, in the pretty.py file where this is used. This should just be the unicode character.

File: sympy/printing/pretty/pretty_symbology.py [hunk 2]
```diff
@@ -580,3 +580,36 @@ def annotated(letter):
...
+     'RightArrow' :                            u"\N{RIGHTWARDS ARROW}",
+     'RightArrowSpaces' :                      u" \N{RIGHTWARDS ARROW} ",
+     'IdenticalTo' :                           u"\N{IDENTICAL TO}",
+     'NotIdenticalTo' :                        u"\N{NOT IDENTICAL TO}",
+     'MultiplicationSign' :                    u"\N{MULTIPLICATION SIGN}",
+     'HorizontalEllipsis':                     u"\N{HORIZONTAL ELLIPSIS}",
+     'SmallElementOf' :                        u"\N{SMALL ELEMENT OF}",
+     'ElementOf' :                             u" \N{ELEMENT OF} ",
+     'DoubleStruckItalicSmallD' :              u"\N{DOUBLE-STRUCK ITALIC SMALL D}",
+     'RightArrowFromBar' :                     u" \N{RIGHTWARDS ARROW FROM BAR} ",
```
Comment (asmeurer) [inline sympy/printing/pretty/pretty_symbology.py]: The leading space should be handled in the pretty.py file.

File: sympy/printing/pretty/pretty_symbology.py [hunk 3]
```diff
@@ -580,3 +580,36 @@ def annotated(letter):
         return ucode_pics[letter]
     else:
         return ascii_pics[letter]
+
+unicode_strings = {
```
Comment (asmeurer) [inline sympy/printing/pretty/pretty_symbology.py]: There are other dictionaries in this file these should go in, as appropriate.

**Comment by ShikharJ** [comment]
@asmeurer I am not confident whether a particular key should be in a particular dictionary or not. I'd rather like to raise an issue for that, after this is merged in, if you don't mind. Can you please take a look?

**Comment by ShikharJ** [comment]
Ping @asmeurer @smichr.

**Comment by ShikharJ** [comment]
Ping @asmeurer

**Comment by asmeurer** [comment]
I would prefer to see them organized, since this creates an API.

**Comment by ShikharJ** [comment]
@asmeurer Frankly I am clueless on where their ideal place should be. None of the other dictionaries, according to me, seem viable for these elements (in the format presently). If you could point out some places, it would be a huge help.

**Comment by ShikharJ** [comment]
Ping @asmeurer.

**Comment by gxyd** [comment]
@ShikharJ is there some reason for why you closed this PR?

**Pull request closed without merging**